package.json

```diff
@@ -1,30 +1,59 @@
 {
   "name": "cookie-policy",
   "version": "1.0.5",
-  "description": "A script and stylesheet that displays the Canonical cookie policy",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "description": "A script and style sheet that displays a cookie policy notification",
+  "main": "index.html",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/canonical-webteam/cookie-policy.git"
   },
-  "author": "Anthony Dillon",
+  "author": "Anthony Dillon <anthony.dillon@canonical.com>",
+  "contributors": [
+    "Barry McGee <barry.mcgee@canonical.com>"
+  ],
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/canonical-webteam/cookie-policy/issues"
   },
   "homepage": "https://github.com/canonical-webteam/cookie-policy#readme",
   "devDependencies": {
-    "babel-preset-es2015": "^6.13.2",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2",
-    "gulp-jshint": "^2.0.1",
-    "gulp-sass": "^2.3.2",
-    "gulp-sass-lint": "^1.2.0",
+    "del": "^3.0.0",
+    "gulp-eslint": "^5.0.0",
+    "gulp-sass-lint": "^1.4.0",
     "gulp-webserver": "^0.9.1",
-    "jshint": "^2.9.2",
-    "vanilla-framework": "^1.6.2"
+    "husky": "^1.3.1",
+    "npm-watch": "^0.6.0",
+    "prettier": "1.16.1",
+    "pretty-quick": "^1.10.0"
+  },
+  "dependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "babel-preset-minify": "^0.5.0",
+    "node-sass": "^4.5.3",
+    "vanilla-framework": "^1.8.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "watch": {
+    "buildJs": "src/js/*.js",
+    "buildScss": {
+      "patterns": [
+        "src/sass"
+      ],
+      "extensions": "scss"
+    }
+  },
+  "scripts": {
+    "clean": "rm -rf build",
+    "test": "eslint ./src/js/**/*.js && sass-lint ./src/sass/**/*.scss",
+    "buildScss": "node-sass ./src/sass/cookie-policy.scss --include-path node_modules --output build/css --output-style compressed",
+    "buildJs": "babel --presets minify ./src/js/cookie-policy.js --out-dir ./build/js/",
```

Comment:
The built JS is also not minified.

Reply:
It is now but it seems Babel doesn't minify templates literals very well....